### ISSUE TITLE
Timeout since 2.3.0.0 with 1Password 8.9

### ISSUE THREAD

**Issue by meilon** [ndbeals/winssh-pageant#25]
Since upgrading to 2.3.0.0 the hangs are now gone, but now timeouts occur on first key use with 1Password 8.9.10. When starting winssh-pageant fresh and I try to connect to a host I get these two messages in the command prompt:

```
Cannot read message size from pipe \\.\pipe\openssh-ssh-agent: i/o timeout
Cannot read message size from pipe \\.\pipe\openssh-ssh-agent: i/o timeout
```

And within PuTTy I get:
```
Authenticating with public key "SSH-Key <key name 1>" from agent
Pageant failed to provide a signature
Authenticating with public key "SSH-Key <key name 2>" from agent
Pageant failed to provide a signature
```

The 1Password window asking for permission to use the key and the subsequent Windows Hello authentication are never waited on as with winssh-pageant 2.2.0.0.

When I allow the use and use biometrics for both keys I can restart the PuTTy session and the connection works with the first key.

**Comment by ndbeals** [comment]
Yeah this was a silly mistake on my part, I set the connection timeout too low, I'll have a push fixed soon

**Comment by ndbeals** [comment]
Fixed in #26, release is published and should be on winget soon
